rskj-core/src/main/java/co/rsk/config/BridgeDevNetConstants.java

```diff
@@ -168,5 +168,8 @@ public BridgeDevNetConstants(List<BtcECKey> federationPublicKeys) {
         maxInputsPerPegoutTransaction = 10;
 
         numberOfBlocksBetweenPegouts = 360; // 3 hours of RSK blocks (considering 1 block every 30 seconds)
+
+        estimatedPegoutTxIndexBtcActivationHeight = 1; // TODO: TBD. This is an estimation of the btc block number once RSKIP379 is activated.
+        pegoutTxIndexGracePeriodInBtcBlocks = 500; // TODO: TBD. Period of grace in btc blocks to continue using the legacy mechanism to identify btc tx.
```

Comment:
Just an idea, maybe we could define this value as something like 
```
btcToRskMinimumConfirmations * 5
```
What do you think? If that's the case we wouldn't need to have it as a Bridge constant since it would only depend on `btcToRskMinimumConfirmations` that already has it's own value for each network

Reply:
I like it. I believe a calculated value using the peg-out minimum is better than creating a constant.

Reply:
Removed.